Nef_3/include/CGAL/Nef_3/bounded_side_3.h

```diff
@@ -30,17 +30,17 @@
 namespace CGAL {
 
 template <class Point_2, class Point_3>
-Point_2 point_3_get_x_y_point_2(Point_3 p) {
+Point_2 point_3_get_x_y_point_2(const Point_3& p) {
```

Comment:
Would it be worthwhile to use `Projection_traits_xy_3` et al?

Reply:
It would be worthwhile not to use `hw()`  for Cartesian coordinates (as you did in another place.